src/inputType/PanInput.ts

```diff
@@ -188,7 +192,11 @@ export class PanInput implements IInputType {
 	*/
 	destroy() {
 		this.disconnect();
-		if (this.hammer && this.hammer.recognizers.length === 1) {
+		if (
+			this.hammer &&
+			this.hammer.recognizers.length === 1 &&
+			this.hammer.recognizers[0] === this.panRecognizer
```

Comment:
@happyhj Why is it needed to check `panRecognizer` exist although `disconnect` already removes `panRecognizer`.